word_way/scrapping/word.py

```diff
@@ -15,13 +17,16 @@
 __all__ = 'save_word',
 
 
-def save_word(target_word: str, session: Session) -> None:
+def save_word(target_word: str,
+              session: Session) -> typing.Optional[uuid.UUID]:
```

Comment:
```suggestion
def save_word(
    target_word: str, session: Session,
) -> typing.Optional[uuid.UUID]:
```
개인적인 취향이지만 이런 줄바꿈은 어떠신가유

Reply:
좋아유

Reply:
@zoripong 반영했어욤

Reply:
확인했어유